workflow/react-project/src/components/common/PasswordInput/index.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+
+import { faEyeSlash } from '@fortawesome/free-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import './style.less';
+
+const PasswordInput = ({ title, errorsMessage, ...inputProps }) => {
+  const [isPasswordVisible, setPasswordVisible] = useState(false);
+
+  const onPasswordIconClick = () => setPasswordVisible(!isPasswordVisible);
+
+  return (
+    <div className="authentication-password-input-wrapper">
+      <p className="authentication-subtitle">{title}</p>
+      <div className="password-input-wrapper">
+        <input className="password-input" type={isPasswordVisible ? 'text' : 'password'} {...inputProps}></input>
+        <FontAwesomeIcon icon={faEyeSlash} className="password-icon" onClick={onPasswordIconClick} />
+      </div>
```

Comment:
it will be moved to a <PasswordControl /> together with state and toggle func (because it's necessary only for password  input )